tests/h/search/index_test.py

```diff
@@ -382,6 +378,27 @@ def test_it_logs_indexing_status(self, caplog, batch_indexer, factories):
                 assert 'indexed 0k annotations, rate=' in record.msg
         assert num_index_records == num_annotations // window_size
 
+    def test_it_correctly_presents_bulk_actions(self, batch_indexer, es_client, factories):
+        anns_a = factories.Annotation.create_batch(3, groupid="group_a")
+        anns_b = factories.Annotation.create_batch(3, groupid="group_b")
+
+        ids_a = [a.id for a in anns_a]
+        ids_b = [a.id for a in anns_b]
+
+        batch_indexer.index()
+
+        for id in ids_a:
```

Comment:
The use of the builtin id will be fixed in a follow up PR as there are other tests that use this.